Makefile

```diff
@@ -255,22 +242,15 @@ $(DMG): nim_status_client $(DMG_TOOL)
 		--identity="NOBODY" \
 		$(MACOS_OUTER_BUNDLE) \
 		pkg || true
-	# `|| true` is used above because code signing will be done manually
-	# below (to allow for MACOS_KEYCHAIN_OPT) but create-dmg doesn't have
-	# an option to not attempt signing. To work around that limitation an
-	# unlikely identity (NOBODY) is specified; this results in a non-zero
-	# exit code even though the .dmg is created successfully (just not code
-	# signed); if the above command failed to create a .dmg then the
-	# following command should result in a non-zero exit code
-	mv "`ls pkg/*.dmg`" pkg/Status.dmg
-
-	# if MACOS_CODESIGN_IDENT is not set then the .dmg is not signed
-	[ -z "$(MACOS_CODESIGN_IDENT)" ] || \
-		codesign \
-			--sign "$(MACOS_CODESIGN_IDENT)" \
-			$(MACOS_KEYCHAIN_OPT) \
-			--verbose=4 \
-			pkg/Status.dmg
+	# We ignore failure above create-dmg can't skip signing.
+	# To work around that a dummy identity - 'NOBODY' - is specified.
+	# This causes non-zero exit code despite DMG being created.
+	# It is just not signed, hence the next command should succeed.
+	mv "`ls pkg/*.dmg`" $(DMG)
```

Comment:
Why the `ls`?

Reply:
I didn't add that.

Reply:
The program that creates the dmg insists on adding a version number to the name of the file. Since there will only be one `.dmg` in there after a build `ls` is a convenient way to get a name we don't know in advance and rename it to `Status.dmg`.

Reply:
You can get the same effect with `mv pkg/*.dmg $(DMG)` - notice the lack of quotes that would prevent the filename globbing.